src/components/Article.jsx

```diff
@@ -1,48 +1,37 @@
 import { useState } from "react";
-
 import { useEffect } from "react";
 import { useParams } from "react-router-dom";
-import { getCommentByArticleID, getSingleArticle } from "../api/APICalls";
+import { getSingleArticle, upVoteArticle } from "../api/APICalls";
+import Comments from "./Comments";
 
 export default function Article() {
   const { article_id } = useParams();
   const [article, setArticle] = useState({});
-  const [comments, setComments] = useState([]);
-
+  const [vote, setVote] = useState(0);
   useEffect(() => {
     getSingleArticle(article_id).then((response) => {
       setArticle(response);
     });
   }, [article_id]);
-
-  useEffect(() => {
-    getCommentByArticleID(article_id).then(
-      (response) => {
-        setComments(response);
-      },
-      [article_id]
-    );
-  });
+  const handleUp = () => {
+    setVote((current_vote) => current_vote + 1);
+    upVoteArticle(article_id, 1).then((response) => {});
+  };
+  const handleDown = () => {
+    setVote((current_vote) => current_vote - 1);
+    upVoteArticle(article_id, -1).then((response) => {});
+  };
```

Comment:
These two functions are very very similar, can you think of a way to combine them so you only have one function used for both up voting and down voting? 

You also have a `.then(...)` which isnt doing anything here, if it dosnt need to do anything the `.then(...)` can and should be removed.